vignettes/GOstatsHyperG.Rmd

```diff
@@ -0,0 +1,521 @@
+---
+title: "How To Use GOstats Testing Gene Lists for GO Term Association"
+author:
+  - name: S. Falcon 
+  - name: R. Gentleman
+  - name: Dennis Ndubi
+    affiliation: "Vignette translation from Sweave to R Markdown / HTML"
+date: "`r format(Sys.time(), '%B %d, %Y')`"
+package: GOstats
+bibliography: GOstats.bib
+link-citations: yes
+vignette: >
+  %\VignetteIndexEntry{Hypergeometric Tests Using GOstats}
+  %\VignetteDepends{ALL, GO.db, annotate, hgu95av2.db, genefilter, RColorBrewer,
+  %xtable, Rgraphviz} 
+  %\VignetteKeywords{Hypergeometric, GO, ontology}
+  %\VignettePackage{GOstats}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+output:
+  BiocStyle::html_document:
+    number_sections: no
+    toc: no
+    toc_depth: 4
+---
+
+# Introduction
+
+The `r Biocpkg("GOstats")` package has extensive facilities for testing
+the association of Gene Ontology (GO) @GO terms to genes in a gene
+list. You can test for both over and under representation of GO terms
+using either the standard Hypergeometric test or a conditional
+Hypergeometric test that uses the relationships among the GO terms for
+conditioning (similar to that presented in Alexa et al. [-@Alexa06]).
+
+In this vignette we describe the preprocessing required to construct
+inputs for the main testing function, `hyperGTest`, the algorithms used,
+and the structure of the return value. We use a microarray data set
+[@Chiaretti2004] from a clinical trial in acute lymphoblastic leukemia
+(ALL) to work an example analysis. In the ALL data, we focus on the
+patients with B-cell derived ALL, and in particular on comparing the
+group with ALL1/AF4 to those with no observed cytogenetic abnormalities.
+
+To get started, load the packages needed for this analysis:
+
+```{r Setup, echo=TRUE, results='hide'}
```

Comment:
Remove `results='hide'` and use `message=FALSE`. If you look at the PDF, you don't see all loading messages that appear in the HTML right now.